src/main/java/com/eternalcode/parcellockers/conversation/ParcelLockerPlacePrompt.java

```diff
@@ -28,8 +24,7 @@ public Prompt acceptInput(@NotNull ConversationContext context, @Nullable String
     @NotNull
     @Override
     public String getPromptText(@NotNull ConversationContext context) {
-        this.announcer.sendMessage((CommandSender) context.getForWhom(), this.config.messages.enterDescriptionPrompt);
-        return StringUtils.EMPTY;
+        return ChatColor.translateAlternateColorCodes('&', this.config.messages.enterDescriptionPrompt);
```

Comment:
Dlaczego to akurat wykorzystuje `translateAlternateColorCodes`?

Reply:
Bo to funkcja Spigotowa która nie wspiera Adventure API